book/src/libs/wasp/benchspy/overview.md

```diff
@@ -0,0 +1,15 @@
+# BenchSpy
+
+BenchSpy (short for Benchmark Spy) is a [WASP](../overview.md)-coupled tool designed for easy comparison of various performance metrics.
+
+## Key Features
+- **Three built-in data sources**:
+  - `Loki`
+  - `Prometheus`
+  - `Direct`
+- **Standard/pre-defined metrics** for each data source.
+- **Ease of extensibility** with custom metrics.
+- **Ability to load the latest performance report** based on Git history.
+- **88% unit test coverage**.
```

Comment:
Let's remove that, it is public docs, not sure people care about that. If they do we have badged for that purpose

Reply:
+

Reply:
👍